helix-term/src/keymap/default.rs

```diff
@@ -200,6 +200,8 @@ pub fn default() -> HashMap<Mode, Keymap> {
             "b" => buffer_picker,
             "s" => symbol_picker,
             "S" => workspace_symbol_picker,
+            "g" => diagnostics_picker,
+            "G" => workspace_diagnostics_picker,
```

Comment:
For comparison, doom emacs have this done is `space c x`, but I don't have much opinion on this key I think.